### PR TITLE
#769 mock service worker in storybook

### PR DESCRIPTION
**Pull request by jojopirker**
I used MSW to mock some API calls (#769). 
The calls for the `/api/valid_labels` endpoint gets mocked for all stories. 
Code is still kinda messy thats why it is a draft, feedback appreciated! :)

### CODE REVIEW

File: website/src/components/Messages/MessageWithChildren.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+};
+
+const Template = (args) => <MessageWithChildren {...args} />;
+
+export const NoChildren = Template.bind({});
+NoChildren.args = {
+  id: "id-1",
+  maxDepth: 2,
+};
+// NoChildren.parameters = {
```
Comment (fozziethebeat) [inline website/src/components/Messages/MessageWithChildren.stories.tsx:38]: Can this be cleaned up?

**Review by fozziethebeat — CHANGES_REQUESTED** [review]
Looks good! Before merging, can you cleanup any commented out code and get the pre-commit check to pass?

**Comment by fozziethebeat** [comment]
@jojopirker Are you able to follow up and finish this PR?  If not, I can copy it and do the finishing touches.

**Comment by jojopirker** [comment]
@fozziethebeat will try to do it tonight!

**Comment by fozziethebeat** [comment]
I had to fork this to get the pre-commit stuff working correctly.

Closing in favor of my replacement PR: https://github.com/LAION-AI/Open-Assistant/pull/967

**Pull request closed without merging**